libclc/CMakeLists.txt

```diff
@@ -151,6 +147,17 @@ if( "spirv-mesa3d-" IN_LIST LIBCLC_TARGETS_TO_BUILD OR "spirv64-mesa3d-" IN_LIST
   endif()
 endif()
 
+if( LIBCLC_TARGETS_TO_BUILD STREQUAL "all" )
```

Comment:
I've put this after the spir-v tool check so that if you already knew to ask for `spirv-mesa3d-`, you will still get an error if the tool is not installed. Instead of getting "unknown target spirv-mesa3d-".

Reply:
Right yeah, because `spirv-mesa3d-` is only added to `ALL` if the tool is available, so would always flag as an unknown target. That makes sense, thanks.